frontend/components/tasks/affectiveAnnotation/inputs/TextfieldWithSeq2Seq.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <v-container class="widget">
+    <v-row class="widget__question" @click="showDialog = true">
+      {{ question }}
+    </v-row>
+    <v-row class="widget__answer" @click="showDialog = true">
+      <v-text-field
+        small
+        dense
+        readonly
+        :value="stringifiedAnswers"
+        :rules="rulesTextfield"
+        hide-details="auto"
+      />
+    </v-row>
+
+    <v-dialog
+      v-model="showDialog"
+      scrollable
+      width="600"
+      class="widget-dialog"
+    >
+      <v-card>
+        <v-card-title class="widget-dialog__title">
+          {{ question }}
+        </v-card-title>
+        <v-card-text  class="widget-dialog__text">
+          <seq2seq-box
+            :text="text"
+            :annotations="answers"
+            @delete:annotation="removeAnswer"
+            @update:annotation="updateAnswer"
+            @create:annotation="addAnswer"
+          />
+        </v-card-text>
+      </v-card>
+    </v-dialog>
+  </v-container>
+</template>
+
+<script>
+import Seq2seqBox from '~/components/tasks/seq2seq/Seq2seqBox'
+
+export default {
+  components: {
+    Seq2seqBox
+  },
+
+  props: {
+    text: {
+      type: String,
+      default: ""
+    },
+    question: {
+      type: String,
+      default: ""
+    },
+    answers: {
+      type: Array,
+      default: () => []
+    },
+    rulesTextfield: {
+      type: Function,
+      default: () => true
+    }
+  },
+
+  data() {
+    return {
+      showDialog: false,
+      stringifiedAnswers: ""
+    }
+  },
+
+  watch: {
+    answers() {
+      const res = []
```

Comment:
I think this can be simplified by using the `map` function, what do you think?

Reply:
Okay, I'm not really used to the map function so my mind keeps thinking with loop. 😅 I will change it